ckan/templates/base.html

```diff
@@ -67,6 +67,9 @@
     #}
     {%- block styles %}
       {% resource g.main_css[6:] %}
+      {% if h.lang() in g.rtl_languages %}
+        {% resource g.rtl_css[6:] %}
```

Comment:
Actually now I see that you'll need two helpers, as you'll also need to get the config value for this (`h.get_rtl_css()`)